src/helpers.py

```diff
@@ -32,14 +32,22 @@ def trim_source(source: str, fn_name: str) -> str:
     return source
 
 
-def load_weights_from_file() -> Mapping[str, Any]:
+def load_weights_from_file(compiler_type: str) -> Mapping[str, float]:
+    weights: Dict[str, float] = {}
     with open("default_weights.toml") as f:
-        return toml.load(f)
+        all_weights = toml.load(f)
```

Comment:
same as I messaged you on discord for the other spot in Main... I can't type this as object without a case

Reply:
"Any means we may have incorrect types here that can inflict the rest of the program and cause errors at a distance."

Well this isn't really true in this case right? and in the other case too.
Because this function returns type Mapping[str, float]... so we know we are getting the correct types there... so wouldn't cause errors at a distance.

I only used Any in locations where the data gets immediately pulled out and type checked and then the Any object is discarded.